inc/cleantalk-common.php

```diff
@@ -613,8 +613,10 @@ function apbct_get_sender_info()
             ? json_encode(Cookie::get('ct_has_input_focused'))
             : null,
         'cache_plugins_detected' => $cache_plugins_detected,
+        //bot detector data
         'bot_detector_fired_form_exclusions' => apbct__bot_detector_get_fired_exclusions(),
         'bot_detector_prepared_form_exclusions' => apbct__bot_detector_get_prepared_exclusion(),
+        'bot_detector_frontend_data_log' => apbct__bot_detector_get_fd_log(),
```

Comment:
No need to collect `apbct__bot_detector_get_fd_log` if the option `Use Anti-Spam by CleanTalk JavaScript library` is disabled.

Reply:
https://github.com/CleanTalk/wordpress-antispam/pull/509/commits/745a32454619fe68606cba10c4fff5704d25d508